### PR TITLE
Fix/validation condition bug

### PR DESCRIPTION
**Pull request by Matt-Roden**
# Description drawing

- Fixed the validation bug
- Moved validation conditional into addPokemon
- Changed the order of operations in Search.js to clear the error and then push through the queriedPokemon

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [x] console.log()
- [x] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -28,14 +28,17 @@ const App = () => {
 
   const addPokemon = (queriedPokemon) => {
     const foundPokemon = validatePokemonQuery(queriedPokemon)
+    if(foundPokemon === undefined) {
+      setError('Not  a valid Name or id , try again')
+    } else {
```
Comment (joanafbrito) [inline src/components/App/App.js:33]: 🤗

File: src/components/PokemonDetails/PokemonDetails.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ import React, { useState, useEffect } from "react";
 import "./PokemonDetails.css";
 
 const PokemonDetails = ({ foundPokemon, getPokemonImage }) => {
+console.log(foundPokemon, 'foundOne<><>')
+
```
Comment (joanafbrito) [inline src/components/PokemonDetails/PokemonDetails.js]: should we delete the console.logs?

**Review by joanafbrito — CHANGES_REQUESTED** [review]

File: src/components/App/App.js (hunk 1 above)
Comment (hayleyw7) [reply src/components/App/App.js:33]: It works!

File: src/components/App/App.js [hunk 3]
```diff
@@ -28,14 +28,17 @@ const App = () => {
...
       setFoundPokemon([foundPokemon])
+    }
   };
 
-// 👇 Only working for a valid query, breaks page on invalid query
   const validatePokemonQuery = (queriedPokemon) => {
     const lowerCaseInput = queriedPokemon.toLowerCase();
 
-    const verifiedName = pokeDex.find((pokemon, index) => {
+     return pokeDex.find((pokemon, index) => {
```
Comment (hayleyw7) [inline src/components/App/App.js:41]: Good fix.

**Review by hayleyw7 — APPROVED** [review]
Great bug fix. 10/10 amazing. So proud!

**Review by joanafbrito — APPROVED** [review]
Cool! Great job team!!

**Pull request merged**